grep-0001-coding-guidelines.md

```diff
@@ -0,0 +1,122 @@
+# GREP [XXXX] -- [Title]
```

Comment:
not to be picky, but set an example by filling the template :)

Reply:
Whoops.